meeting-notes/2023.05.30_meeting_notes.md

```diff
@@ -0,0 +1,87 @@
+
+# OpenJS Foundation Standards Working Group Meeting 2023-05-30
+
+## Links
+
+* **Recording**: https://youtu.be/yUq3M-W8Zy8
+* **GitHub Issue**: https://github.com/openjs-foundation/standards/issues/234
+
+## Present
+
+- Jory Burson (@jorydotcom)      
+- Lea Verou (@leaverou) 
+- Leo Cunha
+- Jordan Harband (@ljharb) 
+- Joe Sepi (@joesepi)
+- Robin Ginn (@rginn) 
+- Tobie Langel (@tobie) 
+- Michael Champion (@michaelchampion)
+- Darcy Clarke (@darcyclarke) 
+- Sendil Kumar (@sendilkumarn) 
```

Comment:
Were Leo Cunha and Sendil Kumar actually present in the meeting? Looking at the recording I didn't see them unless I'm missing something, did they join after the recording ended?

Reply:
(Unrelated but I just noticed that I'm not a member of the standards WG even though I'm an active member in the README, is that intentional?)

Reply:
Thanks, I just updated the team to match the readme.

Reply:
Leo was certainly present as he was explicitly welcomed; I assume that Sendil was as well altho I don't think they spoke?

Reply:
Ah I see, though yeah not sure about Sendil, maybe @jorydotcom can give an insight?

Reply:
Sendil joined late but did not speak. By default YouTube won't show all the participants in the recording - just the ones who spoke.

Reply:
Ah thank you for the insight!